clients/admin-ui/src/features/integrations/configure-monitor/ConfigureWebsiteMonitorForm.tsx

```diff
@@ -79,7 +79,7 @@ const ConfigureWebsiteMonitorForm = ({
   const initialValues: WebsiteMonitorConfig = {
     name: monitor?.name || "",
     execution_frequency:
-      monitor?.execution_frequency || MonitorFrequency.MONTHLY,
+      monitor?.execution_frequency || MonitorFrequency.NOT_SCHEDULED,
```

Comment:
i believe we want to default website monitors to be unscheduled, confirming with product/design separately...

Reply:
confirmed in https://ethyca.atlassian.net/browse/HA-520?focusedCommentId=49345